h/static/styles/partials-v2/_search.scss

```diff
@@ -174,9 +201,26 @@
     }
 }
 
+// As the screen size gets smaller, just before it would have introduced a
+// horizontal scrollbar move the sidebar below the search results.
+@media screen and (max-width: $tablet-width + 225px) {
+  .search-result-container {
+    flex-direction: column;
+    margin-top: 25px;
+  }
+
+  .search-result-nav {
+    display: flex;
+  }
+
+  .search-result-hide-on-small-screens {
+    display: none;
+  }
+}
+
 // On large tablets and below left-align annotation cards in order to make more
 // space for the stats to the right of the annotation card.
-@media screen and (max-width: $tablet-width + 150px) {
+@media screen and (max-width: 1127px) {
```

Comment:
I guess I don't really get the point of using `$tablet-width` here, given that we're then going to have to add an arbitrary hardcoded value to it. 1127px is the exact right width at which this needs to come into effect